cmd/mountlib/help.go

```diff
@@ -261,6 +261,19 @@ Mounting on macOS can be done either via [macFUSE](https://osxfuse.github.io/)
 FUSE driver utilizing a macOS kernel extension (kext). FUSE-T is an alternative FUSE system
 which "mounts" via an NFSv4 local server.
 
+#### macFUSE Notes
+
+If installing macFUSE using [dmg packages](https://github.com/osxfuse/osxfuse/releases) from 
+the website, rclone will locate the macFUSE libraries without any further intervention.  
+If however, macFUSE is installed using the [macports](https://www.macports.org/) package manager, 
+the following addition steps are required.
+
+```bash
+sudo mkdir /usr/local/lib
+cd /usr/local/lib
+sudo ln -s /opt/local/lib/libfuse.2.dylib
```

Comment:
Did you miss the `.` for the destination here?

Reply:
Nope.  If no destination argument, then it is assumed the filename of the first.  See second form of GNU ln man page.  Same syntax for BSD ln.